marimbabot_vision/scripts/data_augmentation.py

```diff
@@ -0,0 +1,50 @@
+from generate_data import NUM_SAMPLES, NUM_WORKER, OUTPUT_DIR
+from multiprocessing import Pool
+from PIL import Image
+
+import os
+import tqdm
+import shutil
+import albumentations as A
+import numpy as np
+import cv2
+
+AUGMENT_OUTPUT_DIR = "data_augmented"
+
+def apply_transforms(img, transforms):
+    """apply given transformations"""
+    transform = A.Compose(transforms)
+       
+    transformed = transform(image=np.asarray(img))
+    transformed_image = transformed["image"]
+
+    return transformed_image
+
+def augment_sample(i, amount = 1):
+    """Generate a augmentations for a given sample and save it to disk"""
+    orig_img_path = f"{OUTPUT_DIR}/{i}/staff_1.png"
+    orig_txt_path = f"{OUTPUT_DIR}/{i}/staff_1.txt"
+    orig_ly_path = f"{OUTPUT_DIR}/{i}/staff_1.ly"
+    img = Image.open(orig_img_path)
+
+    # set of simpler transformations
+    transformations = [ 
```

Comment:
The transformations can be defined once at the top of the file so they are not instanciated every time and it easier to see what is applied.